.github/workflows/build_with_conda.yml

```diff
@@ -16,6 +16,12 @@ jobs:
       matrix:
         include:
           - os: ubuntu-latest
+            cpp_tests: "1"
+            python_tests: "0"
```

Comment:
Nit: Would `run_cpp_tests` and `run_python_tests` be clearer?

Reply:
I have no strong opinion.